tailwind.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  theme: {
+    extend: {
+      fontFamily: {
+        sans: ['Lato', 'sans-serif']
```

Comment:
Import `const defaultTheme = require('tailwindcss/defaultTheme')` above and change this to:

```suggestion
        sans: [
          'Lato',
          ...defaultTheme.fontFamily.sans,
        ]
```

That way you have all the fallbacks from the default Tailwind config.